### DISCUSSION TITLE
How can I merge Links

### DISCUSSION THREAD

**Discussion by asasn** [nocodb/nocodb#7482] [General]
I have multiple columns in a row, each with some links, and I need to combine all the Links in the last column. How do I do that?
If I extract {filed_1}+{filed_2} using Formula, I get only the number of Links statistics.

**Comment by mertmit** [comment]
Hi @asasn,
check our URL function in formula: https://docs.nocodb.com/fields/field-types/formula/string-functions#url
So you can achieve the result you want using: `URL({filed_1}+{filed_2})`
